eng/targets/CSharp.Common.props

```diff
@@ -13,7 +13,7 @@
   </ItemGroup>
 
   <ItemGroup Condition=" '$(IsTestProject)' != 'true' AND '$(DotNetBuildFromSource)' != 'true' ">
-    <Reference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" PrivateAssets="All" />
+    <Reference Include="Microsoft.CodeAnalysis.NetAnalyzers" PrivateAssets="All" />
```

Comment:
I'm not sure what is this reference supposed to do when it's already in `Dependencies.props`